### PR TITLE
Add Cqrs, deleted services

### PR DESCRIPTION
**Pull request by BogdanChernikov**

### CODE REVIEW

File: Controllers/QuizController.cs [hunk 1]
```diff
@@ -1,41 +1,36 @@
...
         {
             return await _mediator.Send(new QuizSummariesQuery());
         }
 
-		[HttpGet]
-		[Route("{quizId}/details")]
-		public async Task<EditQuizViewModel> GetQuizDetails([FromRoute] int quizId)
+        [HttpGet]
+        [Route("{quizId}/details")]
+        public async Task<QuizDetailsQueryResult> GetQuizDetails([FromRoute] int quizId)
```
Comment (VolodymyrMaryniak) [inline Controllers/QuizController.cs]: Use QuizDetailsQuery as a parameter for this action. Use [FromRoute] attribute for Id property.

File: Cqrs/Commands/Handlers/UpdateQuizCommandHandler.cs [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+            // Save changes
+            if (_quizRepository.HasChanges())
+            {
+                quizEntity.ModifiedDate = DateTime.Now;
+                await _quizRepository.UpdateAsync();
+            }
+
+            return new UpdateQuizCommandResult { Success = true };
+        }
+        private void UpdateQuestion(QuestionEntity questionEntity, Question question)
```
Comment (VolodymyrMaryniak) [inline Cqrs/Commands/Handlers/UpdateQuizCommandHandler.cs]: Add space between methods. Make method static

File: Cqrs/Commands/Handlers/UpdateQuizCommandHandler.cs [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+                    UpdateAnswerOption(existingAnswerOption, answerOption);
+                }
+                else
+                {
+                    questionEntity.AnswerOptions.Add(CreateAnswerOption(answerOption));
+                }
+            }
+        }
+
+        private QuestionEntity CreateQuestion(Question question)
```
Comment (VolodymyrMaryniak) [inline Cqrs/Commands/Handlers/UpdateQuizCommandHandler.cs]: Make method static

File: Cqrs/Commands/Handlers/UpdateQuizCommandHandler.cs [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+                AnswerOptions = question.AnswerOptions.Select(CreateAnswerOption).ToList()
+            };
+        }
+
+        private static void UpdateAnswerOption(AnswerOptionEntity answerOptionEntity, AnswerOption answerOption)
+        {
+            answerOptionEntity.IsCorrectAnswer = answerOption.IsCorrectAnswer;
+        }
+
+        private AnswerOptionEntity CreateAnswerOption(AnswerOption answerOption)
```
Comment (VolodymyrMaryniak) [inline Cqrs/Commands/Handlers/UpdateQuizCommandHandler.cs]: Make method static

File: Cqrs/Queries/QuizDetailsQuery.cs [hunk 5]
```diff
@@ -0,0 +1,10 @@
+﻿using AspNetCoreVueStarter.Cqrs.Queries.Results;
+using MediatR;
+
+namespace AspNetCoreVueStarter.Cqrs
```
Comment (VolodymyrMaryniak) [inline Cqrs/Queries/QuizDetailsQuery.cs]: Change namespace

File: Cqrs/Queries/Results/QuizDetailsQueryResult.cs [hunk 6]
```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace AspNetCoreVueStarter.Cqrs.Queries.Results
+{
+    public class QuizDetailsQueryResult
```
Comment (VolodymyrMaryniak) [inline Cqrs/Queries/Results/QuizDetailsQueryResult.cs:6]: Do not change property names. Rollback to previous names.)

**Review by VolodymyrMaryniak — CHANGES_REQUESTED** [review]

**Comment by VolodymyrMaryniak** [comment]
Revert changes for file: ClientApp/package-lock.json

File: ClientApp/package-lock.json [hunk 7]
```diff
@@ -1,8 +1,17650 @@
 {
   "name": "asp-net-core-vue-starter",
```
Comment (VolodymyrMaryniak) [inline ClientApp/package-lock.json]: Revert changes for this file: ClientApp/package-lock.json

File: Controllers/QuizController.cs [hunk 8]
```diff
@@ -1,41 +1,36 @@
-﻿using AspNetCoreVueStarter.Cqrs.Commands;
+﻿using AspNetCoreVueStarter.Cqrs;
```
Comment (VolodymyrMaryniak) [inline Controllers/QuizController.cs]: Remove not used models.

File: Controllers/QuizController.cs [hunk 9]
```diff
@@ -1,41 +1,36 @@
...
+using AspNetCoreVueStarter.Cqrs.Commands;
 using AspNetCoreVueStarter.Cqrs.Commands.Results;
```
Comment (VolodymyrMaryniak) [inline Controllers/QuizController.cs:2]: Remove using of QuizSummaryViewModel

**Review by VolodymyrMaryniak — CHANGES_REQUESTED** [review]

**Comment by BogdanChernikov** [comment]
Done

File: Controllers/QuizController.cs (hunk 8 above)
Comment (BogdanChernikov) [reply Controllers/QuizController.cs]: Done

File: ClientApp/package-lock.json (hunk 7 above)
Comment (BogdanChernikov) [reply ClientApp/package-lock.json]: Done

File: Controllers/QuizController.cs (hunk 1 above)
Comment (BogdanChernikov) [reply Controllers/QuizController.cs]: Done

File: Cqrs/Queries/Results/QuizDetailsQueryResult.cs (hunk 6 above)
Comment (BogdanChernikov) [reply Cqrs/Queries/Results/QuizDetailsQueryResult.cs:6]: Done

**Review by VolodymyrMaryniak — APPROVED** [review]

**Pull request merged**